tests/nodeos_eos_evm_test.py

```diff
@@ -109,13 +113,13 @@ def interact_with_storage_contract(dest, nonce):
             nonce=nonce,
             gas=100000,       #100k Gas
             gasPrice=gasP,
-            to=Web3.toChecksumAddress(dest),
+            to=Web3.to_checksum_address(dest),
```

Comment:
Looks like we are using `web3` python package v6.

We should update the package installation step in the `PERFORMANCE.md` doc.

```
...
pip install 'web3<6' flask flask-cors
...
```

Reply:
Addressed: [f7c496a](https://github.com/eosnetworkfoundation/eos-evm-node/pull/17/commits/f7c496acbf04de703de6f45a136863882eff94a6)